Gemfile

```diff
@@ -11,7 +11,7 @@ gem 'coffee-rails'
 gem 'uglifier'
 gem 'jquery-rails', '~> 4.0.4'
 gem 'jbuilder'
-gem 'turbolinks', github: 'rails/turbolinks'
+gem 'turbolinks', git: 'https://github.com/rails/turbolinks.git'
```

Comment:
By default, bundler-audit consider host like `git://` and `http://` is insecure, Bundler provided `github` syntax will expand to `git://github.com/rails/turbolinks.git`.

Either use a released version of turbolink or https can fix this.